File: app_watcher/src/main/java/com/tongchen/twatcher/mzitu/view/IMZiTuCategoryView.java
package com.tongchen.twatcher.mzitu.view;

import com.tongchen.twatcher.base.view.IMVPView;
import com.tongchen.twatcher.mzitu.model.entity.MZiTu;

import java.util.List;

/**
 * Created by TongChen at 11:02 on 2018/12/28.
 * <p>
 * Description: 妹子图按分类获取封面列表
 */
public interface IMZiTuCategoryView extends IMVPView<List<MZiTu>> {

    void noMoreData();

}

File: app_watcher/src/main/java/com/tongchen/twatcher/mzitu/view/IMZiTuCategoryView.kt
package com.tongchen.twatcher.mzitu.view

import com.tongchen.twatcher.base.view.IMVPView
import com.tongchen.twatcher.mzitu.model.entity.MZiTu

/**
 * @author TongChen
 * @date 2019/11/28  15:38
 * <p>
 * Desc: 妹子图按分类获取封面列表
 */
interface IMZiTuCategoryView : IMVPView<MutableList<MZiTu>> {

    fun noMoreData()
}